analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -89,11 +90,13 @@ class Cargo(
     private fun extractVcsInfo(node: JsonNode) =
         VcsHost.toVcsInfo(extractRepositoryUrl(node))
 
-    private fun extractDeclaredLicenses(node: JsonNode) =
-        node["license"].textValueOrEmpty().split("/")
+    private fun extractDeclaredLicenses(node: JsonNode): SortedSet<String> {
+        val licenses = node["license"].textValueOrEmpty().split('/')
             .map { it.trim() }
             .filter { it.isNotEmpty() }
-            .toSortedSet()
+
+        return if(licenses.isEmpty()) sortedSetOf<String>() else sortedSetOf(licenses.joinToString(" OR "))
```

Comment:
This should be (note the space after `if`, and the removed explicit type):

    if (licenses.isEmpty()) sortedSetOf() else sortedSetOf(licenses.joinToString(" OR "))

Reply:
Is the case distinction actually needed, e.g. why not just `sortedSetOf(licenses.joinToString(" OR "))`?

Reply:
Because that would create a `SortedSet` containing an empty string if `licenses` is empty, and not the wanted empty set.

Reply:
Oh right, fully missed that.

Reply:
> `if (licenses.isEmpty()) sortedSetOf() else sortedSetOf(licenses.joinToString(" OR "))`

When I tried that, it didn't compile, I can try again, though.

Reply:
I also just tried, for me it compiles fine without the explicit type.

Reply:
done.